### ISSUE TITLE
Accessibility (major): Landing page should contain a level 1 header [argo-001]

### ISSUE THREAD

**Issue by aaron-collier** [sul-dlss/argo#4160]
Change the "Welcome to Argo!" header from `<h3>` to `<h1>`. Ideally, the Welcome message and instructions would not appear within the search results content container, but as the first element in the `<main>` container. This would require a more significant redesign of the page.

Currently, "Welcome to Argo!" is an `h1` with a class of `h3`. This approach was used to avoid styling conflicts with recent a11y work but may be insufficient.

https://docs.google.com/spreadsheets/d/1m3obE89_mBCZpRkWAjJAXEr2ZfrUirzjvsJWqYH-G6Y/edit#gid=0

**Comment by lwrubel** [comment]
Since as you noted, "Welcome" is now an `<h1>`, is the task to use a different class than `h3`? Or to figure out how to change the aria-label on the `<main>` container to not be `"search results"` before there are search results?

**Comment by mjgiarlo** [comment]
Question for you, @aaron-collier :point_up:

**Comment by mjgiarlo** [comment]
@aaron-collier Hiven the page already has an `<h1>` header, and we don't have a new design for the landing page, it wasn't entirely clear to me what I could do with this issue. This is the best I came up with: https://github.com/sul-dlss/argo/pull/4191
